packages/config-loader/src/loader.ts

```diff
@@ -67,9 +67,6 @@ export type LoadConfigOptions = {
   // Paths to load config files from. Configs from earlier paths have lower priority.
   configTargets: ConfigTarget[];
 
-  /** @deprecated This option has been removed */
-  env?: string;
```

Comment:
Note to self: need to remove configuration options from `dist-types/packages/config-loader/src/loader.d.ts`.

Reply:
Bonus note: that's generated by `yarn tsc` and should probably not be modified manually? :p

Reply:
Wha! Gotcha -- thanks for clarifying.